sdk/pinocchio/src/program_error.rs

```diff
@@ -3,7 +3,7 @@
 //! Current implementation is based on the `ProgramError` enum from
 //! the Solana SDK:
 //!
-//! https://github.com/anza-xyz/agave/blob/master/sdk/program/src/program_error.rs
+//! <https://github.com/anza-xyz/agave/blob/master/sdk/program/src/program_error.rs>
```

Comment:
May as well update this link to point to the sdk repo since it's no longer correct :smile: